csharp/src/Ice/Internal/PropertiesAdminI.cs

```diff
@@ -166,32 +166,14 @@ public override void
                 // Copy callbacks to allow callbacks to update callbacks
                 foreach (var callback in new List<Ice.PropertiesAdminUpdateCallback>(_deprecatedUpdateCallbacks))
                 {
-                    try
-                    {
-                        callback.updated(changes);
-                    }
-                    catch (System.Exception ex)
```

Comment:
I just removed this error handling. The user-supplied callback should not throw any exception, and if it does, it will get logged via Ice.Warn.Dispatch / Ice.Trace.Dispatch.